extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/interceptor/AbstractMethodValidationInterceptor.java

```diff
@@ -101,7 +107,8 @@ protected void validateConstructorInvocation(InvocationContext ctx) throws Excep
         ctx.proceed();
         Object createdObject = ctx.getTarget();
 
-        violations = validator.forExecutables().validateConstructorReturnValue(ctx.getConstructor(), createdObject);
+        violations = validatorInstance.get().forExecutables().validateConstructorReturnValue(ctx.getConstructor(),
```

Comment:
Noticed while working on a similar patch and a good opportunity to fix it:
```suggestion
        violations = executableValidator.validateConstructorReturnValue(ctx.getConstructor(),
```